formalmethods_intrepid_app_upload_102.py

```
@@ -0,0 +1,36 @@
+    try:
+        parser = Parser()
+        ctx = parser.parse_file(filepath)
+        name = '__ctx{}'.format(len(contexts))
+        contexts[name] = {'context': ctx,
+                          'inputs': ctx.inputs,
+                          'nets': ctx.nets,
+                          'latches': ctx.latches,
+                          'engines': {},
+                          'traces': {},
+                          'simulators': {}}
+        return {'result': {'ctx': name}}, 201
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)